disable_odoo_online/data/modules.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <!-- delete enterprise modules -->
```

Comment:
This should go in the other module: res_config_settings_enterprise_remove

Reply:
You still haven't removed this.

Reply:
Because

> I think this part could kept here

:blush: 

What do others think?
@hbrunn @StefanRijnhart @astirpe @legalsylvain @hieulucky111

Reply:
I think delete tags are harmful because last time I checked they break if the record is already removed. And the fact that enterprise modules are present in the list of modules after an Odoo installation is less relevant to this module as they do not cause any traffic to Odoo by themselves I think.

Reply:
@pedrobaeza @StefanRijnhart done